packages/opc-feedback/src/opc-feedback.ts

```diff
@@ -502,6 +502,7 @@ export class OpcFeedback extends LitElement {
             ${this.docs &&
             html`<li>
               <a
+                target="_blank"
```

Comment:
@sshamsudRH This should not be a default action, add a check and pass it down from the component attribute. There might be a use-case where we what this to open in the same page.